testdata/dnn/download_models.py

```diff
@@ -39,23 +39,23 @@ def verify(self):
             return False
         print('  expect {}'.format(self.sha))
         sha = hashlib.sha1()
-        with open(self.filename, 'rb') as f:
-            while True:
-                buf = f.read(self.BUFSIZE)
-                if not buf:
-                    break
-                sha.update(buf)
-        print('  actual {}'.format(sha.hexdigest()))
-        return self.sha == sha.hexdigest()
-
-    def get(self):
         try:
-            if self.verify():
-                print('  hash match - skipping')
-                return True
+            with open(self.filename, 'rb') as f:
+                while True:
+                    buf = f.read(self.BUFSIZE)
+                    if not buf:
+                        break
+                    sha.update(buf)
+            print('  actual {}'.format(sha.hexdigest()))
```

Comment:
Lets print debug information on errors only:
```
if self.sha != sha.hexdigest():
    print('  expect {}'.format(self.sha))
    print('  actual {}'.format(sha.hexdigest()))
```

Perhaps it make sense to add separate messages `if not os.path.exists(self.filename):`